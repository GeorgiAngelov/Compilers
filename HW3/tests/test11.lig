/* This is a multi-line comment
**/
function main(argv : [[int]]) : int {
/* Another
***/
/*** A third
 *
 *
 ***/
      return (0);
}
