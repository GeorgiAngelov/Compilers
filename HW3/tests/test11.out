(decls
   (decl-fun:ok (id main) (type-fun (params (typed-id (id argv) (type-array (type-array (type-int))))) (type-int))
      (decls)
      (stmts
         (return:ok (num:int 0)))))
